api-update/build.sbt

```diff
@@ -0,0 +1,69 @@
+import Dependencies._
+import sbt.Keys.fork
+
+ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "uk.gov.nationalarchives"
+ThisBuild / organizationName := "api-update"
+
+
+lazy val root = project
+  .in(file("."))
+  .settings(commonSettings)
+  .disablePlugins(AssemblyPlugin)
+  .aggregate(
+    common
+  )
+
+lazy val common = project
+  .settings(
+    name := "common",
+    commonSettings,
+    testSettings,
+    libraryDependencies ++= commonDependencies
+  )
+  .disablePlugins(AssemblyPlugin)
+
+lazy val antivirus = project
+  .settings(
+    name := "antivirus",
+    commonSettings,
+    assemblySettings,
+    testSettings,
+    libraryDependencies ++= commonDependencies
+  )
+  .dependsOn(
+    common % "test->test;compile->compile"
```

Comment:
Is the `compile` bit necessary? I'd have thought it would automatically compile a dependent project.

Just checked, and we didn't have to do this in the prototype Sangria project: https://github.com/nationalarchives/tdr-prototype-sangria/blob/master/build.sbt#L65 But it was a bit different, because we didn't have tests.

Reply:
They're both necessary. If I take either one out, I can't run assembly without errors. Maybe it's because they're both sub-projects whereas sangria had a root project and a sub-directory project. That's just a guess mind. Anyway, they're both needed.